pkg/services/ngalert/api/provisioning.go

```diff
@@ -28,6 +28,10 @@ func (f *ProvisioningApiHandler) handleRouteGetContactpoints(ctx *contextmodel.R
 	return f.svc.RouteGetContactPoints(ctx)
 }
 
+func (f *ProvisioningApiHandler) handleRouteGetContactpointsExport(ctx *contextmodel.ReqContext) response.Response {
```

Comment:
Consider adding rate limiting or some form of protection against potential abuse of the export endpoint. This could help prevent potential denial-of-service attacks or excessive resource usage. [important]